tools/mdm/migration/echo/main.go

```diff
@@ -19,9 +23,11 @@ func main() {
 			detail = fmt.Sprintf("| BODY: %s", string(body))
 		}
 		log.Printf("%s %s %s\n", request.Method, request.URL.Path, detail)
-		if err := request.Write(writer); err != nil {
-			log.Fatalf(err.Error())
+		if _, err := writer.Write(nil); err != nil {
+			log.Printf("error writing response %s", err.Error())
 		}
+
+		time.Sleep(DELAY)
```

Comment:
You may want to move the delay before calling `writer.Write` to simulate a slow response, as it may flush the whole response on that call otherwise.